CHANGES.rst

```diff
@@ -7,6 +7,10 @@ git master
 New features
 ''''''''''''
 
+* End of build summary of failing examples with traceback. It is
```

Comment:
Summary of failing examples with traceback at the end of the sphinx build. By default the build exits with a 1 exit code as soon as at least one example is failing. A list of examples that are expected to fail can be defined in conf.py.